src/Common/Utils.fs

```diff
@@ -0,0 +1,20 @@
+module Common.Utils
+
```

Comment:
This is so hardcore that I would probably just create a hardcoded sequence of pronouns here and run `Seq.random` on it... But if you like this kind of stuff, I will understand :D Code-wise this is probably as best as it can take to be generic, in that case I would just suggest moving `getRandomUnion` to `Common.Utils` and moving `getRandomPronoun` to `Verb`.